probe-rs/targets/esp32c2.yaml

```diff
@@ -50,14 +50,15 @@ flash_algorithms:
     pc_init: 0x0
-    pc_uninit: 0x56
-    pc_program_page: 0x3c
-    pc_erase_sector: 0x2a
-    pc_erase_all: 0x34
-    data_section_offset: 0x4038C07C
+    pc_uninit: 0x1d4
+    pc_program_page: 0x5a
+    pc_erase_sector: 0x48
+    pc_erase_all: 0x52
+    data_section_offset: 0x40396d64
     load_address: 0x4038C000
+    transfer_encoding: Miniz
```

Comment:
Nit: we might want to make this serialize/deserialize as lowercase to be consistent with the rest of the yaml?